pio/ir_nec/ir_loopback/ir_loopback.c

```diff
@@ -0,0 +1,67 @@
+/**
+ * Copyright (c) 2021 mjcross
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#include <stdio.h>
+#include "pico/stdlib.h"
+
+#include "nec_transmit.h"                           // include the library headers
+#include "nec_receive.h"
+
+// Infrared loopback example ('NEC' format)
+//
+// Need to connect an IR LED to GPIO 14 via a suitable series resistor (e.g. 1.5k)
+// and an ective-low IR detector to GPIO 15 (e.g. VS1838b)
```

Comment:
ective-low -> active-low ?

Reply:
yes that's a typo thanks

Reply:
fixed with https://github.com/raspberrypi/pico-examples/pull/129/commits/0162c73b0164ef07f71a68f326f4140781ae25d0